app/lib/getNearbyServices.js

```diff
@@ -1,23 +1,43 @@
-const log = require('../../app/lib/logger');
+const log = require('../lib/logger');
 const esClient = require('./esClient');
-const filterServices = require('./filterServices');
+const addMessages = require('./addMessages');
+const getDateTime = require('./getDateTime');
 const VError = require('verror').VError;
 const esGetPharmacyHistogram = require('./promHistograms').esGetPharmacy;
+const addMessagesHistogram = require('../lib/promHistograms').addMessages;
 
 async function getNearbyServices(searchCoordinates, limits) {
   let endTimer;
-  let pharmacies;
+  let nearbyPharmacies;
+  let openPharmacies;
 
   try {
     endTimer = esGetPharmacyHistogram.startTimer();
-    pharmacies = await esClient.getPharmacies(searchCoordinates, limits.searchRadius);
+    openPharmacies = await esClient.getOpenPharmacies(
+      getDateTime(),
+      searchCoordinates,
+      limits.searchRadius,
+      limits.open
+    );
+    nearbyPharmacies = await esClient.getPharmacies(
+      searchCoordinates,
+      limits.searchRadius,
+      limits.nearby
+    );
   } catch (err) {
     log.error({ err: new VError(err) });
     throw err;
   } finally {
     endTimer();
   }
-  return filterServices(pharmacies, limits);
+
+  const fiterTimer = addMessagesHistogram.startTimer();
+  const results = {
+    nearbyServices: addMessages(nearbyPharmacies),
```

Comment:
Within the call to `addMessages` there is call to `getDateTime`. This will return a different result between `nearbyServices` and `openServices` which will also be different to the time used on [line 17](https://github.com/nhsuk/nearby-services-api/pull/101/files#diff-6b3c91a372d0bb755710f47f6b10c6e3R17).
I'd suggest using the same value for all three function calls else inconsistent results may occur around time boundaries.